eo-maven-plugin/src/test/java/org/eolang/maven/ResolveMojoTest.java

```diff
@@ -85,11 +85,11 @@ void resolveWithoutAnyDependencies(@TempDir final Path temp) throws IOException
             .set(AssembleMojo.ATTR_VERSION, "0.22.1");
         final Path target = temp.resolve("target");
         this.resolve(new DummyCentral(), foreign, target);
-        final Path path = temp.resolve("target/06-resolve/org.eolang/eo-runtime/-/0.28.11");
+        final Path path = temp.resolve("target/06-resolve/org.eolang/eo-runtime/-/");
         MatcherAssert.assertThat(path.toFile(), FileMatchers.anExistingDirectory());
         MatcherAssert.assertThat(
-            path.resolve("eo-runtime-0.28.11.jar").toFile(),
-            FileMatchers.anExistingFile()
+            path,
+            new ContainsFile("**/*.jar")
```

Comment:
@mximp I believe it would be better to use more precise pattern, like `**/eo-runtime-*.jar`, because the test can give us false-positive, if we add some other dependency.

Reply:
@volodya-lombrozo corrected